cogs/ncplayz.py

```diff
@@ -0,0 +1,51 @@
+import discord
+from discord.ext import commands
+
+class NCPlayz:
+    def __init__(self, bot):
+        self.bot = bot
+        
+    @commands.command(aliases = ["id"])
+    async def ID(self, ctx):
+        """Get the ID of a channel, user, or role"""
+        if ctx.invoked_subcommand is None:
+            await ctx.send('Invalid Type! {0.subcommand_passed} is not a valid command!'.format(ctx))
+
+    @ID.command()
+    async def channel(self, ctx, *, channel: discord.Channel = None):
+        """Fetches the ID of a specified channel"""
+        channel = channel or ctx.channel
+        if channel:
+            msg = "The id of the channel `{0}` is `{0.id}".format(channel)
+        else: 
+            msg = "**Error!** A channel named {0} could not be found! You must enter the exact name (including caps).".format(request)
+        await ctx.send(msg)
+        
+
+    @ID.command()
+    async def role(self, ctx, *, role: discord.Role):
+        """Fetches the ID of a specified role"""
+        if role:
+            msg = "The id of the role `{0}` is `{0.id}".format(role)
+        else: 
+            msg = "**Error!** A role named {0} could not be found! You must enter the exact name (including caps).".format(request)
+        await ctx.send(msg)
+    
+    @ID.command()
+    async def member(self, ctx, *, member: discord.Member = None):
+        """Fetches the ID of a specified member/user."""
+        member = member or ctx.author
+        if member:
+            msg = "The id of the member `{0.name}` is `{0.id}".format(member)
+        else:
+            msg = "**Error!** A member named {0} could not be found! You must enter the exact name (including caps).".format(request)
+        await ctx.send(msg)
+
+    @ID.command()
+    async def server(self, ctx):
+        """Fetches the ID of the server """
```

Comment:
be consistent, you give a choice (default value) in the rest of the commands but omit it in server.